src/vic3_world/world/vic3_world_importer.cpp

```diff
@@ -272,6 +272,28 @@ void AssignMilitaryFormationsToCountries(const std::map<int, vic3::MilitaryForma
 }
 
 
+void ApplySubjectRelationships(const std::map<int, vic3::Pact>& pacts, std::map<int, vic3::Country>& countries)
+{
+   for (const vic3::Pact& pact: pacts | std::views::values)
+   {
+      if (pact.isSubjectRelationship())
+      {
+         auto overlord = countries.find(pact.GetFirstId());
```

Comment:
I had an issue with this type of logic many times when either of the countries doesn't survive the conversion due to being an opm or similarly squeezed out of maproom. Unless the relationship doesn't matter, you should definitely have a check on HoI4 side and delete relationships referencing dead (landless) countries post relationship transfer.